form.py

```diff
@@ -0,0 +1,59 @@
+from flask_wtf import Form
+from wtforms import StringField, BooleanField, SelectField, TextAreaField, RadioField
+from flask_wtf.file import FileField
+from wtforms.validators import DataRequired, Email, URL
+from werkzeug import secure_filename
+
+class ApplicationForm(Form):
+    name = StringField('Name', validators=[DataRequired()])
```

Comment:
Errors aren't yet being communicated to the user. I'm cool with that being outside the scope of this PR, but let's get 'em in before launch.